### ISSUE TITLE
Project meta field size

### ISSUE THREAD

**Issue by kevinpapst** [mxgross/EncryptedProjectFieldBundle#3]
The field in Kimai is only 256 chars long, not 512.
https://github.com/mxgross/EncryptedProjectFieldBundle/blob/master/EventSubscriber/MetaFieldSubscriber.php#L41
Is the size decreased by the encryption?

**Comment by mxgross** [comment]
Good point. I have to check it. Could you prefer a size for the field. It should be possible to add a lot of text there.

**Comment by kevinpapst** [comment]
But meta fields were not made for that purpose.
Maybe use the new comment fields? They have an unlimited text size

**Comment by mxgross** [comment]
Seems to be a better idea, I did not know about this comment fields yet. I will see what I can make as soon I find the time to do it :)
